documentation/commands/git-town.md

```diff
@@ -51,5 +52,15 @@ git town version
 > git town perennial-branches --remove qa
 > ```
 
+* *pull-branch-strategy*
+> Displays the pull branch strategy. (default: 'rebase')
```

Comment:
I would remove the period here (this is not a proper sentence), and add an empty like after this line.